net/squid4/Portfile

```diff
@@ -133,6 +133,22 @@ if {![variant_isset gnutls]} {
     default_variants    +openssl
 }
 
+variant sslbump conflicts gnutls description "Enable SSL Bump support" {
+    default_variants-append +openssl
+    configure.args-append   --enable-ssl-crtd
+    configure.args-append   --enable-http-violations
+    configure.args-append   --enable-security-cert-validators
+    configure.args-append   --enable-security-cert-generators
+    configure.args-append   --enable-follow-x-forwarded-for
+    configure.args-append   --enable-forw-via-db
+    configure.args-append   --disable-eui
```

Comment:
This can be disabled in the config, but it looks like it doesn't work on macOS anyway, so might as well always use this option?